cli/src/test-utils/test-logger.ts

```diff
@@ -1,38 +1,54 @@
 import chalk from "chalk";
 
-export const TestLogger = {
-  error,
-  log,
-  success,
-  warn
-};
-
-// tslint:disable:no-console
-function error(message: string, opts?: LoggerOpts) {
-  console.log(chalk.red(transformMessage(message, opts)));
-}
+export class TestLogger {
+  /** Prepares an object for printing */
+  static formatObject(obj: any): string {
+    return JSON.stringify(obj, undefined, 2);
```

Comment:
what's 2?

Reply:
the number of spaces to use for json indentation
```
2 space:
{
  "blah": 123
}

10 space:
{
          "blah": 123
}
```

Reply:
TIL